src/platform/silabs/efr32/CHIPCryptoPALPsaEfr32.cpp

```diff
@@ -43,6 +43,19 @@
 #include <string.h>
 #include <type_traits>
 
+// Includes needed for certificate parsing
```

Comment:
I am assuming this is mbedtls code since I have seen some of this elsewhere in the repo, so I am skipping it. Am I right?

If so, since this lives in our files, maybe it would be worthy to identify the parts that are EFR32 specific? To shorten the review time and to allow for someone else to pickup this file in the future if need be?

Reply:
These functions are actually part of the CHIPCryptoPALPSA.h interface. They were implemented in the old CHIPCryptoPALPsaEfr32.cpp, but are missing in CHIPCryptoPALPSA.cpp. As far I know, devices don't use them, but the test driver fails to compile without them. I suspect there is an error in the GN dependencies, but fixing them would expand the scope of this PR.